tests/lib/bin/mountinfo-tool

```diff
@@ -749,7 +749,8 @@ Known attributes, applicable for both filtering and display.
         raise SystemExit(
             "--one requires exactly one match, found {}".format(len(entries))
         )
-
+    if len(entries) == 0:
+        raise SystemExit(1)
```

Comment:
How about making the output describe why it fails? Something like `raise SystemExit("no mount entry found for given input")` (or something along the inlines, maybe we should even include the inputs?

Reply:
I kind of like the "unix tools don't talk much" approach. I'm not opposed to adding this behind a verbose flag. This PR was motivated by a test cleanup that used `mountinfo-tool` as a test condition so I don't technically need the verbosity here.